database/sqlite/sqlite_functions.c

```diff
@@ -313,6 +337,7 @@ void delete_dimension_uuid(uuid_t *dimension_uuid)
             error_report("Failed to prepare statement to delete a dimension uuid");
             return;
         }
+        add_stmt_to_list(res);
```

Comment:
```
        rc = sqlite3_prepare_v2(db_meta, DELETE_DIMENSION_UUID, -1, &res, 0);
        if (rc != SQLITE_OK) {
            error_report("Failed to prepare statement to delete a dimension uuid");
            return;
        }
        add_stmt_to_list(res);
```
This 4 lines keep repeating... I would probably make function netdata_sqlite_prepare_v2 or sth. that ensures correct handling

Reply:
Yes I will improve on a next iteration

Reply:
Fixing it